backend/core/src/units-summary/entities/units-summary.entity.ts

```diff
@@ -34,13 +34,13 @@ class UnitsSummary {
   @Column({ nullable: true, type: "integer" })
   @Expose()
   @IsOptional({ groups: [ValidationsGroupsEnum.default] })
-  @IsNumberString({}, { groups: [ValidationsGroupsEnum.default] })
+  @IsNumber({}, { groups: [ValidationsGroupsEnum.default] })
```

Comment:
do you need to re-run the backend-swagger generation for this?

Reply:
It didn't generate anything, this is just a validation decorator, it doesn't change the actual type.